.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: python
 python:
+  - "3.9"
+  - "3.8"
+  - "3.7"
   - "3.6"
   - "3.5"
-  - "pypy3"
```

Comment:
Any reason to drop PyPy3?

Reply:
No specific reasons. I removed as I haven't tested with it.

Reply:
Please add it back and we'll eventually move to Github actions at some point.